dart/dynamics/Shape.h

```diff
@@ -121,7 +126,7 @@ class Shape : public virtual common::Subject
   //           biased mesh shape. Two Vector3ds might be better; one is for
   //           minimum verterx, and the other is for maximum verterx of the
   //           bounding box.
-  const Eigen::Vector3d& getBoundingBoxDim() const;
+  Eigen::Vector3d getBoundingBoxDim() const;
```

Comment:
`DEPRECATED(5.2)` would be fine. In general, we might not be sure the deprecation is included in which release at the moment. So I usually increment the minor version with 1 and update the version number if needed when DART is released.